invenio_requests/resources/requests/resource.py

```diff
@@ -54,6 +73,23 @@ def search(self):
         )
         return hits.to_dict(), 200
 
+    @request_extra_args
+    @request_search_args
+    @request_view_args
+    @response_handler(many=True)
+    def search_user_requests(self):
+        """Perform a search over user requets.
+
+        /GET /user/requests
+        """
+        hits = self.service.search_user_requests(
+            identity=g.identity,
+            params=resource_requestctx.args,
+            es_preference=es_preference(),
+            expand=resource_requestctx.args["expand"],
```

Comment:
Can you please change this to:
```suggestion
            expand=resource_requestctx.args.get("expand", False),
```